src/app/shared/feature.service.ts

```diff
@@ -12,5 +12,7 @@ export class FeatureService {
     const gitHubAppCallBackToNewDashBoard = urlSearchParams.has('state') && urlSearchParams.get('state').includes('newDashboard');
     const newDashboard = urlSearchParams.has('newDashboard') || gitHubAppCallBackToNewDashBoard;
     Dockstore.FEATURES.enableNewDashboard = newDashboard;
+    const notebooks = urlSearchParams.has('notebooks');
+    Dockstore.FEATURES.enableNotebooks = notebooks;
```

Comment:
Not a huge deal, but you could have something similar to lines 12/13... It handles the case of:

1. You specify notebooks as a query param, notebooks are visible.
2. You register a new entry which takes you to GitHub apps
3. You come back to Dockstore, and notebooks are no longer visible.